Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -523,10 +524,10 @@ private async Task<string> PutObjectAsync(string bucketName, string objectName,
         {
             var queries = new List<string>();
             queries.Add("uploads");
-            if (prefix != null)
-            {
-                queries.Add("prefix=" + Uri.EscapeDataString(prefix));
-            }
+            queries.Add("prefix=" + Uri.EscapeDataString(prefix));
```

Comment:
Same comment as above regarding possible `null` values passed for `delimiter` and `prefix` arguments.